src/build.py

```diff
@@ -1130,99 +1134,112 @@ def DebianPackage():
 
 
 def CompileLLVMTorture(extension, outdir):
-  name = 'Compile LLVM Torture (%s)' % extension
-  buildbot.Step(name)
-  c = Executable(os.path.join(INSTALL_BIN, 'clang'))
-  cxx = Executable(os.path.join(INSTALL_BIN, 'clang++'))
-  Remove(outdir)
-  Mkdir(outdir)
-  unexpected_result_count = compile_torture_tests.run(
-      c=c, cxx=cxx, testsuite=GCC_TEST_DIR,
-      sysroot_dir=INSTALL_SYSROOT,
-      fails=LLVM_KNOWN_TORTURE_FAILURES,
-      out=outdir,
-      config='wasm-' + extension)
-  UploadArchive('torture-' + extension, Archive(outdir))
-  if 0 != unexpected_result_count:
-    buildbot.Fail()
+  for opt in BARE_TEST_OPT_FLAGS:
```

Comment:
Although I guess the downside would be that we'd have 3 sets of `for opt in flags: LinkLLVMTorture()` if we wanted to keep the same test ordering that we have here, which I think we do. Maybe that's not so bad though.